### PR TITLE
Distance_3: Add Pl_3-Pl_3

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes
The squared_distance(Plane_3, Plane_3) implementation was missing
## Release Management

* Affected package(s):Distance_3

### CODE REVIEW

File: Distance_3/include/CGAL/squared_distance_3_2.h [hunk 1]
```diff
@@ -399,6 +399,21 @@ squared_distance(const Triangle_3<K> & t,
 }
 
 
+template <class K>
+inline
+typename K::FT
+squared_distance(const Plane_3<K> & p1,
+                 const Plane_3<K> & p2) {
+  K k;;
```
Comment (sloriot) [inline Distance_3/include/CGAL/squared_distance_3_2.h]:
```suggestion
  K k;
```

File: Distance_3/include/CGAL/squared_distance_3_2.h [hunk 2]
```diff
@@ -399,6 +399,21 @@ squared_distance(const Triangle_3<K> & t,
...
+  if( k.compute_squared_length_3_object()(
```
Comment (sloriot) [inline Distance_3/include/CGAL/squared_distance_3_2.h]:
```suggestion
  typename K::Construct_orthogonal_vector_3 ortho_vec = k.construct_orthogonal_vector_3_object();
  if (is_null(wcross(ortho_vec(p1), ortho_vec(p2), k), k))
```

File: Distance_3/include/CGAL/squared_distance_3_2.h [hunk 3]
```diff
@@ -399,6 +399,21 @@ squared_distance(const Triangle_3<K> & t,
...
+        k.construct_cross_product_vector_3_object()(p1.orthogonal_vector(),
```
Comment (sloriot) [inline Distance_3/include/CGAL/squared_distance_3_2.h]:
```suggestion
```

File: Distance_3/include/CGAL/squared_distance_3_2.h [hunk 4]
```diff
@@ -399,6 +399,21 @@ squared_distance(const Triangle_3<K> & t,
...
+                                                    p2.orthogonal_vector()))
```
Comment (sloriot) [inline Distance_3/include/CGAL/squared_distance_3_2.h]:
```suggestion
```

File: Distance_3/include/CGAL/squared_distance_3_2.h [hunk 5]
```diff
@@ -399,6 +399,21 @@ squared_distance(const Triangle_3<K> & t,
...
+      != 0)
```
Comment (sloriot) [inline Distance_3/include/CGAL/squared_distance_3_2.h]:
```suggestion
```

File: Distance_3/test/Distance_3/test_distance_3.cpp [hunk 6]
```diff
@@ -221,6 +221,17 @@ struct Test {
...
   }
 
+  void Pl_Pl()
+  {
+    std::cout << "Plane - Plane\n";
+    Pl p1(12, 1, 4, 0);
+    typename K::Vector_3 v = -p1.orthogonal_vector();
+    v /= CGAL::sqrt(v.squared_length());
+    Pl p2 = Pl(p1.point()+6*v, -p1.orthogonal_vector());
+    check_squared_distance (p1,p2, 36);
```
Comment (sloriot) [inline Distance_3/test/Distance_3/test_distance_3.cpp]: I'm surprised this works, you better construct yourself a plane, rather rely on inexact sqrt.

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-183.shtml

**Pull request merged**